src/Controls/src/Core/Registrar.cs

```diff
@@ -11,7 +11,8 @@ namespace Microsoft.Maui.Controls
 	[Flags]
 	public enum InitializationFlags : long
 	{
-		DisableCss = 1 << 0
+		DisableCss = 1 << 0,
+		SkipRenderers = 1 << 1,
 	}
 
 	// Previewer uses reflection to bind to this method; Removal or modification of visibility will break previewer.
```

Comment:
well, this can go away now? No more Previewer

Reply:
Gone!